src/components/flow/actions/localization/MsgLocalizationForm.tsx

```diff
@@ -233,7 +233,7 @@ export default class MsgLocalizationForm extends React.Component<
 
     if (typeConfig.localizeableKeys!.indexOf('quick_replies') > -1) {
       tabs.push({
-        name: 'Quick Replies',
+        name: i18n.t('form.quick_replies', 'Quick Replies'),
```

Comment:
I see `forms.*` and `form.*` I think this is supposed to be `forms.quick_replies`. I wouldn't be shocked if there are others.